src/lib.rs

```diff
@@ -27,6 +32,8 @@ pub fn robyn(_py: Python<'_>, m: &PyModule) -> PyResult<()> {
     m.add_class::<FunctionInfo>()?;
     m.add_class::<PyRequest>()?;
     m.add_class::<PyResponse>()?;
+    m.add_class::<MiddlewareType>()?;
+    m.add_class::<HttpMethod>()?;
```

Comment:
@AntoineRR , do we need to export this `HttpMethod` from rust?

Reply:
Yes because I changed the code to use this enum when asking Rust to add a route (we were relying on an `str` before my changes)

Reply:
ah, okay. makes sense :D